gitlint/cli.py

```diff
@@ -122,14 +122,15 @@ def stdin_has_data():
               help="Verbosity, more v's for more verbose output (e.g.: -v, -vv, -vvv). [default: -vvv]", )
 @click.option('-s', '--silent', help="Silent mode (no output). Takes precedence over -v, -vv, -vvv.", is_flag=True)
 @click.option('-d', '--debug', help="Enable debugging output.", is_flag=True)
```

Comment:
`--debug` should be either removed or mapped to `loglevel=debug`.

Reply:
This breaks even more tests and is not what we want to achieve here. Rejected.